sig-node/sig-node-contributor-ladder.md

```diff
@@ -0,0 +1,116 @@
+
+- Reviewed PRs must be merged.
+- Since the purpose is to demonstrate the nominee's technical depth, PRs like analyzer warnings fixes, mechanical “find/replace”-type PRs, minor improvements of kubelet logging and insignificant bug fixes are valued, but not counted towards the reviewer status nomination. Lack of reviews of those PRs may be a red flag for nomination approval.
+- Making the most of reviews in a single area is a good indication that a subdirectory reviewer role may be preferred or top-level node reviewer.  Finding the right balance helps balance the nominees future workload.
+- A primary reviewer should drive the review of the PR without significant input / guidance from the approver or other reviewers. 
+
+**Has enough context**
+
+It is hard to assess codebase knowledge and it always will be a judgement call. SIG Node will rely on the listed PR to ensure the person reviewed PRs from different areas of SIG Node codebase and on the comments made during SIG Node meetings.
+
+Additional ways to establish the knowledge of context are:
+- Contributions to k8s documentation
+- Blog posts - k8s-hosted and external
+- Talks at conferences and meetups
+- Contributions to other SIGs
+
+**Trustworthy**
+
+Reviewer nominations are accepted by SIG Node approvers. SIG Node approvers take nominations seriously and are invested in building a healthy community.  Nominees should help approvers understand their future goals in the community so we can help continue to build trust and mutual relationships and nurture new opportunities if and when a contributor wants to become an approver!
+
+### Approver
+
+SIG Node approvers have a lot of responsibilities and it is a very demanding role as the project keeps expanding in capability. It is expected that the SIG Node approver keeps the codebase quality high by giving feedback, thoroughly reviewing code, and giving recommendations to SIG members and reviewers.  SIG Node approvers are essentially gatekeepers to keep the code base at high quality. SIG Node maintains a rigidly high bar for becoming a SIG Node approver by developing trust in a community and demonstrating expertise with a bias towards initial code contributions over reviewing PRs.
+
+SIG Node approver role is not based on a volume of ongoing work, there is no absolute quota. Approver role is based on accumulated trust and knowledge and it is not expected to require 100% full time commitment from any individual member. The expectation is that an active (non emeritus) approver is responsive to a direct ask to review a complex PR or KEP in their area of expertise when solicited.  It is encouraged for a top-level approver to opt out of a review where another approver is more appropriate.
```

Comment:
Do we want to handle the emeritus approvers handling here as well? At which point do we consider an approver as emeritus?

Reply:
@saschagrunert i prefer to keep the current PR on how to grow scope.

Reply:
Sounds good!